### PR TITLE
Do not generate JavaDocs for empty comments #280

### LINKED ISSUE #280: Summary module generates empty comments. Should be no comment in this case.
**Description**

There can be a situation where summary is a non-empty list with blank characters, like: regularStmt("\t\n").
As a result a comment is blank.
In this case the comment should not be shown.

This is a description of a probable bug which is known for many tests.

### PR DESCRIPTION
**Pull request by onewhl**
# Description

Do not generate JavaDocs for empty comments. 
Fixes #280

## Type of Change

Please delete options that are not relevant.

- Minor bug fix (non-breaking small changes)

# How Has This Been Tested?

## Automated Testing

Run tests in summary-tests module.

## Manual Scenario 
Run it on the example provided in the issue #280

# Checklist (remove irrelevant options):

- [ ] The change followed the style guidelines of the UTBot project
- [x] Self-review of the code is passed
- [ ] The change contains enough commentaries, particularly in hard-to-understand areas
- [ ] New documentation is provided or existed one is altered
- [ ] No new warnings
- [ ] New tests have been added
- [x] All tests pass locally with my changes

### CODE REVIEW

File: utbot-summary/src/test/kotlin/org/utbot/summary/comment/SimpleClusterCommentBuilderTest.kt [hunk 1]
```diff
@@ -48,15 +48,14 @@ class SimpleClusterCommentBuilderTest {
     fun `builds empty comment if execution result is null`() {
         val commentBuilder = SimpleClusterCommentBuilder(traceTag, sootToAst)
         val comment = commentBuilder.buildString(sootMethod)
-        assertEquals(" ", comment)
+        assertEquals("", comment)
```
Comment (amandelpie) [inline utbot-summary/src/test/kotlin/org/utbot/summary/comment/SimpleClusterCommentBuilderTest.kt:51]: Add constant empty string?

File: utbot-summary/src/test/kotlin/org/utbot/summary/comment/SimpleClusterCommentBuilderTest.kt [hunk 2]
```diff
@@ -48,15 +48,14 @@ class SimpleClusterCommentBuilderTest {
...
 
     @Test
-    fun `builds empty doc statement if execution result is null`() {
+    fun `does not build any statements for javadoc if execution result is null`() {
         val commentBuilder = SimpleClusterCommentBuilder(traceTag, sootToAst)
         val statements = commentBuilder.buildDocStmts(sootMethod)
-        assertEquals(statements.size, 1)
-        assertEquals(statements[0].toString(), " ")
+        assertEquals(statements.size, 0)
     }
```
Comment (amandelpie) [inline utbot-summary/src/test/kotlin/org/utbot/summary/comment/SimpleClusterCommentBuilderTest.kt:60]: Remove empty line

**Review by amandelpie — APPROVED** [review]
Please fix minor comments

File: utbot-summary/src/test/kotlin/org/utbot/summary/comment/SimpleClusterCommentBuilderTest.kt (hunk 1 above)
Comment (onewhl) [reply utbot-summary/src/test/kotlin/org/utbot/summary/comment/SimpleClusterCommentBuilderTest.kt]: Thanks, done!

File: utbot-summary/src/test/kotlin/org/utbot/summary/comment/SimpleClusterCommentBuilderTest.kt (hunk 2 above)
Comment (onewhl) [reply utbot-summary/src/test/kotlin/org/utbot/summary/comment/SimpleClusterCommentBuilderTest.kt]: Done!

**Pull request merged**